apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/functions/NotificationFunctions.java

```diff
@@ -99,4 +99,34 @@ public HttpResponseMessage resendNotification(
         notificationEventService.send(context, onboarding.get(), queueEvent);
         return request.createResponseBuilder(HttpStatus.OK).build();
     }
+
+    /**
+     * This HTTP-triggered function sends messages through event hub.
+     * It gets invoked by module onboarding-cdc when status is COMPLETED or DELETED
+     */
+    @FunctionName("NotificationTest")
+    @FixedDelayRetry(maxRetryCount = 3, delayInterval = "00:00:30")
+    public HttpResponseMessage sendNotificationTest (
+            @HttpTrigger(name = "req", methods = {HttpMethod.POST}, authLevel = AuthorizationLevel.FUNCTION) HttpRequestMessage<Optional<String>> request,
+            final ExecutionContext context) {
+        context.getLogger().info("sendNotifications trigger processed a request");
+
+        String onboardingString = request.getBody().orElseThrow(() -> new IllegalArgumentException("Request body cannot be empty."));
+        final String queueEventString = request.getQueryParameters().get("queueEvent");
+        final QueueEvent queueEvent = Objects.isNull(queueEventString) ? null : QueueEvent.valueOf(queueEventString);
+        final Onboarding onboarding;
+
+        try {
+            onboarding = readOnboardingValue(objectMapper, onboardingString);
```

Comment:
In HttpTrigger you can pass an POJO type directly instead to parse from a string.... so you could pass Onboarding type on HttpRequestMessage